_data/file_formats/eff_v2/body.yml

```diff
@@ -13,16 +13,24 @@
   type: dword
 - desc: |
     Target type
-    - 0=None
-    - 1=Self (pre-projectile)
-    - 2=Pre-target
-    - 3=Party
-    - 4=Everyone (inc. party)
-    - 5=Everyone (excl. party)
-    - 6=Everyone matching specific value of caster (or Party if cast by party member)
-    - 7=Everyone matching specific value of target
```

Comment:
These two bits of info shouldn't be removed, as it truly checks the specifics stat.

Reply:
> These two bits of info shouldn't be removed, as it truly checks the specifics stat.

Yeah, this is mostly WIP, still gathering more info about that...

Reply:
What do you want to know, perhaps I can help?

Reply:
> What do you want to know, perhaps I can help?

I should copy-paste what I wrote for V1 Effects (SPL/ITM) since it's valid here too... And then that additional note explains when/if this field is relevant or not...

Reply:
I don't understand why anyone would think these two values would be useless — they clearly allow pretty fine-grained filtering. IWD-style ids matching is another, but of course not available in all games and not quite the same.

Reply:
> I don't understand why anyone would think these two values would be useless — they clearly allow pretty fine-grained filtering. IWD-style ids matching is another, but of course not available in all games and not quite the same.

OK, I'll specify that the `SPECIFIC.IDS` thing is EE only and keep the removed part for non-EE games...

Reply:
Wait, why? Everyone uses the specifics stat, that's my whole point ...

Reply:
> Wait, why? Everyone uses the specifics stat, that's my whole point ...

OK, so what are suggesting exactly? Could you directly propose file change?

Reply:
I would just use the old description, additionally linking to the CRE offset for specifics.

Reply:
> I would just use the old description, additionally linking to the CRE offset for specifics.

Since this field is only relevant when called via op249, I've simply added a link to V1 effects (all details are there, including the old description and link to  CRE offset for specifics...)